### PR TITLE
feat: added rulesets

### PR DESCRIPTION
**Pull request by tstaetter**
needed for node firewalls

### CODE REVIEW

File: blockjoy/api/v1/node.proto [hunk 1]
```diff
@@ -181,13 +181,25 @@ message Rule {
   Policy policy = 2;
```
Comment (mjr-blockjoy) [inline blockjoy/api/v1/node.proto:181]: lets rename `Policy` to `Action`

File: blockjoy/api/v1/node.proto [hunk 2]
```diff
@@ -181,13 +181,25 @@ message Rule {
...
   Direction direction = 3;
   optional Protocol protocol = 4;
-  optional string ips = 5;
+  string dns_name = 5;
   repeated uint32 ports = 6;
+  bool enabled = 7;
+  optional Policy default_in = 8;
```
Comment (mjr-blockjoy) [inline blockjoy/api/v1/node.proto:187]: default_in/out and fallback actions when packet doesn't match any rule, so it doesn't make sense to put it into every rule(even optional)

File: blockjoy/api/v1/node.proto [hunk 3]
```diff
@@ -171,23 +172,48 @@ enum Direction {
...
+}
+
+message RuleSet {
+  string id = 1;
+  repeated Rule rules = 2;
+  string org_id = 3;
 }
 
+// CRUD for a whole ruleset
 message NodeFirewallUpdate {
```
Comment (mjr-blockjoy) [inline blockjoy/api/v1/node.proto:210]: This one is used in line 157 (command sent to BV). I think we should always send the same message to BV, both on create and in firewall update command, and id should be `FirewallConfiguration`.

**Review by densone — CHANGES_REQUESTED** [review]
rename ruleset back to policy in all code.

**Pull request closed without merging**